### PR TITLE
feat(): updating to latest natchez and log4cats, tweaking Span defini…

### PR DESCRIPTION
**Pull request by brbrown25**
…tion so that we can honor the api changes.

### CODE REVIEW

File: natchez-ce3/src/main/scala/com/ovoenergy/natchez/extras/combine/ce3/IOLocalTrace.scala [hunk 1]
```diff
@@ -28,4 +28,14 @@ class IOLocalTrace(private val local: IOLocal[Span[IO]]) extends Trace[IO] {
 
   override def traceUri: IO[Option[URI]] =
     local.get.flatMap(_.traceUri)
+
+  override def log(fields: (String, TraceValue)*): IO[Unit] = ???
```
Comment (voidcontext) [inline natchez-ce3/src/main/scala/com/ovoenergy/natchez/extras/combine/ce3/IOLocalTrace.scala:32]: In `src/main` there shouldn't be any unimplemented methods.

**Review by voidcontext — CHANGES_REQUESTED** [review]
Thank you for your contribution!

There are some unimplemented methods in non test classes. These need to be implemented so that `natchez-extras` is fully compliant with `natchez` v0.2.*.

File: natchez-ce3/src/main/scala/com/ovoenergy/natchez/extras/combine/ce3/IOLocalTrace.scala (hunk 1 above)
Comment (brbrown25) [reply natchez-ce3/src/main/scala/com/ovoenergy/natchez/extras/combine/ce3/IOLocalTrace.scala:32]: I'll work on implementing those today!

**Comment by brbrown25** [comment]
@voidcontext I'm not very familiar with this library so I took a swing in the dark on this, any help would be greatly appreciated :)

**Comment by alexcardell** [comment]
natchez 0.3.0 is now released, will that be saved for a 8.0 release of natchez-extras?

**Comment by voidcontext** [comment]
Closing this in favour of #101

**Pull request closed without merging**